tests/acceptance/conftest.py

```diff
@@ -109,6 +110,20 @@ def project(gl: Gitlab, group: Group):
     gitlab_project.delete()
 
 
+@pytest.fixture(scope="class")
+def project_hook(gl: Gitlab, project):
+    name = get_random_name("hook")
+    hook = project.hooks.create({"url": f"http://hooks/{name}.org"})
+
+    yield hook
+    # When using in hook deletion test this is Not Found
+    try:
+        hook.delete()
+    except (GitlabDeleteError, GitlabHttpError) as e:
+        if e.response_code == 404:
+            pass
```

Comment:
Is it possible something else is going wrong here? Didn't think it's expected to return 404 when trying to delete a hook.

Reply:
It happened whenever I tested deleting a hook - first it got yielded from here, I use it and delete it in my test, then when it got back to here after its scope,  to remove the yielded hook, it was now gone (at least, that's what I thought was happening). 
When using the http calls directly, when a delete gets called it comes with "accepted_codes" including 404, so I thought passing this would be ok.